edxval/tests/test_api.py

```diff
@@ -334,6 +334,43 @@ def test_get_videos_for_ids_duplicates(self):
         videos = list(api.get_videos_for_ids([edx_video_id, edx_video_id]))
         self.assertEqual(len(videos), 1)
 
+    def test_get_videos_for_ids_sort(self):
+        fish_id = constants.VIDEO_DICT_FISH["edx_video_id"]
+        star_id = constants.VIDEO_DICT_STAR["edx_video_id"]
+        other_id = "other-video"
+        Video.objects.create(**constants.VIDEO_DICT_STAR)
+        Video.objects.create(
```

Comment:
The constants file is already very crowded, and this video is specifically crafted to sort interestingly compared with the other two. I think that's clearer if it's defined here; I can certainly add a comment to that effect.